File: Java/URI/uri1036.java
import java.util.Scanner;

public class uri1036{
	public static void main(String[] args){
		Scanner scanner = new Scanner(System.in);
		double a = scanner.nextDouble();
		scanner.nextLine();
		double b = scanner.nextDouble();
		scanner.nextLine();
		double c = scanner.nextDouble();
		scanner.nextLine();

		float delta = (float)(b*b - (4*a*c));

		if(delta > 0){
		double r1 = (double)((-b + Math.sqrt(delta)) / (2.0 * a));
		double r2 = (double)((-b - Math.sqrt(delta)) / (2.0 * a));
		System.out.format("R1 = %.5f\nR2 = %.5f", r1, r2);}
		else{
			System.out.println("Impossivel calcular");
		}
	}
}

File: Python/URI/uri1036.py
print "URI 1036"
from math import * 
A, B, C = [float(i) for i in raw_input("").split(" ")]
delta = B**2-(4*A*C)
base = 2*A
if delta < 0 or base == 0:
    print "Impossivel calcular"
else:
	R1 = (-B + sqrt(delta))/base
	R2 = (-B - sqrt(delta))/base
	print "R1 = %.5f" % R1
	print "R2 = %.5f" % R2
